README.md

```diff
@@ -134,24 +134,39 @@ Usage:
 - `abc templates golden-test record [--test-name=<test_name>] [<location>]`
 - `abc templates golden-test verify [--test-name=<test_name>] [<location>]`
 
+Note: for `new-test`, the `<location>` parameter gives the location of the template.
+for `record` and `verify`,  the `<location>` parameter gives the location that include one or more templates.
+
 Examples:
 
 - `abc templates golden-test new-test basic examples/templates/render/hello_jupiter`
   creates a new golden-test for the specific named test called `basic` for the
   given template
-- `abc templates golden-test verify examples/templates/render/hello_jupiter`:
+- `abc templates golden-test verify examples/templates/render/hello_jupiter`
```

Comment:
Could we add examples for `abc templates golden-test record` with no parameters? That would mean "record all tests underneath current directory".

And same for `verify` please.